Edited Step 2 in README

TheOnAndOnlyZenomat opened the pull request:
Ignore this, I am working on some stuff and forgot something

TheOnAndOnlyZenomat committed:
Edited Step 2 in README

Changed the REAME Point 2, so that it is apparent, that you have to specify the path to you wallpaperfolder. Because of this the first step was obsolete and therefore I removed it.

BogdanTheGeek commented:
What are you working on? I forgot I added the argument for extprimcol before making the script public.
Thanks for letting me know

TheOnAndOnlyZenomat commented:
I am working on implementing feh as a wallpaper setter and I fixed an issue with the formating in the myprimcolors.txt which prevented the script from working on my end. I also added the functionality, that there is no need for a specific folder. I am almost finished, give me half an hour, then I'll create another pull request.
